src/MessageHandler.py

```diff
@@ -0,0 +1,36 @@
+import urllib2
+from src.CastsCommandHandler import CastsCommandHandler
+from src.MessageRouter import MessageRouter
+from src.StreamsCommandHandler import StreamsCommandHandler
+
+
+class MessageHandler:
+    """
+    This class is responsible for constructing the object graph needed
+    to handle messages and to delegate to it.
+    """
```

Comment:
This class could be expressed essentially as a function, which `__init__` being a static configuration and `handle_message` being the equivalent of `__call__`